README.md

```diff
@@ -13,30 +11,40 @@ Devices connected to IoT Hub can de described with DTDL, in terms of:
 - Commands (Using the Hub Direct Methods feature)
 - Telemetry (processed through EventHub)
 
-Review the docs [IoT Plug and Play Preview Documentation](https://review.docs.microsoft.com/azure/iot-pnp/?branch=release-preview-refresh-iot-pnp).
+Review the docs in the private staging environment [IoT Plug and Play Preview Documentation](https://review.docs.microsoft.com/azure/iot-pnp/?branch=pr-en-us-114283)
 
-### Private environments, and preview SDKs
+### BugBash support and feedback
 
-As the time of writing the version of IoT Hub supporting PnP BugBash 2, 08 May2020 release is only deployed to a private hub instance:
+- The teams channel [PnP Public Preview BugBash](https://teams.microsoft.com/l/channel/19%3a0b9d0f166a3d41c69ce90fcca7631962%40thread.skype/PnP%2520Public%2520Preview?groupId=dcc1ac84-f476-4c96-8034-b2d77e54c8bf&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47). Will have team members answering questions during the BugBash.
```

Comment:
@miagdp can we use this teams channel?

Reply:
LGTM, using the same as previous, less confusing